### PR TITLE
Extract planning application creation on a service

### PR DESCRIPTION
**Pull request by EGiataganas**
### Description of change

Extract planning application creation on a service

### CODE REVIEW

File: app/importers/planning_applications_importer.rb [hunk 1]
```diff
@@ -48,21 +48,28 @@ def import_rows(filename:)
...
+      property_type: row[:property_type],
+      full: row[:full],
+      address: row[:address],
+      town: row[:town],
+      postcode: row[:postcode],
+      map_east: row[:map_east],
+      map_north: row[:map_north],
+      ward_code: row[:ward_code],
+      ward_name: row[:ward_name],
+    ).perform
```
Comment (allbecauseyoutoldmeso) [inline app/importers/planning_applications_importer.rb:72]: Would `PlanningApplicationCreation.new(**row.merge(local_authority: local_authority))` work here?

File: app/services/planning_application_creation.rb [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    @town = params.fetch(:town, nil)
+    @postcode = params.fetch(:postcode, nil)
+    @map_east = params.fetch(:map_east, nil)
+    @map_north = params.fetch(:map_north, nil)
+    @ward_code = params.fetch(:ward_code, nil)
+    @ward_name = params.fetch(:ward_name, nil)
+  end
+
+  def perform
+    importer
```
Comment (allbecauseyoutoldmeso) [inline app/services/planning_application_creation.rb:31]: You could just call `#importer` `#perform` instead?

Of alternatively use `#alias_method` here? https://apidock.com/ruby/Module/alias_method

**Review by allbecauseyoutoldmeso — APPROVED** [review]
Looks great!

File: app/importers/planning_applications_importer.rb (hunk 1 above)
Comment (EGiataganas) [reply app/importers/planning_applications_importer.rb:72]: Yeah I could, just thought is better to explicitly pass each attribute. Happy to change it

Comment (EGiataganas) [reply app/importers/planning_applications_importer.rb:72]: I've tried this change and some specs are failing so I will leave it as it is

File: app/services/planning_application_creation.rb (hunk 2 above)
Comment (EGiataganas) [reply app/services/planning_application_creation.rb:31]: The `#alias_method` is not going to work as `#importer` is private. I defined a second public method `#perform` to call it!

**Pull request merged**